goblet/infrastructures/vpcconnector.py

```diff
@@ -0,0 +1,90 @@
+from googleapiclient.errors import HttpError
+from goblet.config import GConfig
+from goblet.infrastructures.infrastructure import Infrastructure
+
+import logging
+
+log = logging.getLogger("goblet.deployer")
+log.setLevel(logging.INFO)
+
+
+class VPCConnector(Infrastructure):
+
+    resource_type = "vpcconnector"
+
+    def register_connector(self, name, kwargs):
```

Comment:
since ipCidrRange is set in config and is required we should check during register if ifCidrRange exists. the other option is to have the user pass the ip range explicity as an argument in the app.vpcconnector decorater